ocs_ci/deployment/helpers/lso_helpers.py

```diff
@@ -91,7 +91,8 @@ def setup_local_storage(storageclass):
         # sleep here to start update machineconfigpool status
         time.sleep(60)
         wait_for_machineconfigpool_status("all")
-
+        if config.DEPLOYMENT.get("ui_deployment", False):
+            return
```

Comment:
Could you add a DEBUG or INFO log message about skipping the rest of the setup because UI deployment was enabled?

Reply:
done